src/extensions/modtools.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, bot):
         inumber = int(number)
         if inumber <= 100:
             # In order to delete the command message too, the number of messages to clear is incremented
+            # noinspection PyUnresolvedReferences

```

Comment:
Oh, that one, for some reason PyCharm is complaining about the return types of coroutines.